src/Explorer/Panes/AddDatabasePane.ts

```diff
@@ -271,6 +271,7 @@ export default class AddDatabasePane extends ContextualPaneBase {
   public open() {
     super.open();
     this.resetData();
+    this.isAutoPilotSelected(this.container.isAutoscaleDefaultEnabled());
```

Comment:
>isAutoPilotSelected [](start = 9, length = 19)

this is not necessary right - since reset data already does this?

Reply:
Yes, I removed the redundant check.